pkg/cloud/rgraph/testing/ez/ez.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package ez is a utility to create complex resource graphs for testing from a
+// concise description by use of naming conventions and default values.
+//
+// This package should only be used for testing. There is little error handling;
+// most errors will result in a panic() to reduce the verbosity of the code.
+package ez
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph"
+)
+
+type GraphOption int
+
+const (
+	// PanicOnAccessErr causes panic() if Accces() returns an error. Otherwise,
+	// the error is ignored.
+	PanicOnAccessErr GraphOption = 1 << iota
+)
+
+func panicf(s string, args ...any) { panic(fmt.Sprintf(s, args...)) }
+
+// Graph is an easy way to build resource Graphs. This is intended for static
+// testing case construction. Errors are handled by panic().
+type Graph struct {
+	// Nodes in the Graph.
+	Nodes []Node
+	// Project to use by default if not specified in the Node.
+	Project string
+	// Options to use by default in the Graph and Nodes.
+	Options GraphOption
```

Comment:
How are multiple options intended to be represented here?

Reply:
bit-wise OR. Let me update the docs

Reply:
It looks like you only added a comment on `NodeOption` but not `GraphOption`